Add week1 content for portfolio page

lolfuljames opened the pull request:
This commit adds content and style to the page, including images and icon needed.

lolfuljames committed:
Add week1 content for portfolio page

henryken commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,291 @@
+    <meta charset="UTF-8" />
+    <title>Home - James Tan</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
```
To improve the website performance better, you can read an article below
https://flaviocopes.com/javascript-async-defer/

lolfuljames replied:
> To improve the website performance better, you can read an article below
> https://flaviocopes.com/javascript-async-defer/

Noted, I'll be adding defer along with some of the minor tweaks to improve the ergonomics of the website..

henryken reviewed (commented):
@lolfuljames, I have informational comment to improve performance better. See if you can read and understand the article below.
https://flaviocopes.com/javascript-async-defer/

lolfuljames committed:
Add improvements to readability and performance

This commit adds javascript optimisation, adds a larger font-size to improve mobile readibility and adds metadata to improve SEO.

henryken reviewed (approved):
Great work!
